src/multibody/joint/joint-dense.hpp

```diff
@@ -30,6 +30,7 @@ namespace se3
       NQ = _NQ, // pb
       NV = _NV
     };
+    typedef double Scalar;
```

Comment:
Currently in all traits ( for Derived Motions, Constraints, and Joints), Scalar is redefined (always a typedef double Scalar).
We once talked about introducing a typedef float/double Scalar in a file such as math/fwd.h but it remained a hypothesis